float32 distance_sensor1
float32 distance_sensor2
